cmd/blockchaincmd/convert.go

```diff
@@ -388,14 +388,19 @@ func InitializeValidatorManager(
 	if err != nil {
 		return tracked, err
 	}
-	// TODO: replace latest below with sig agg version in flags for convert and deploy
-	err = signatureaggregator.CreateSignatureAggregatorInstance(app, network, aggregatorLogger, "latest")
-	if err != nil {
-		return tracked, err
-	}
-	signatureAggregatorEndpoint, err := signatureaggregator.GetSignatureAggregatorEndpoint(app, network)
-	if err != nil {
-		return tracked, err
+	var signatureAggregatorEndpoint string
+	if signatureAggregatorFlags.SignatureAggregatorEndpoint == "" {
+		// TODO: replace latest below with sig agg version in flags for convert and deploy
+		err = signatureaggregator.CreateSignatureAggregatorInstance(app, network, aggregatorLogger, "latest")
```

Comment:
take into account comments made on the other PR, regarding making GetSignatureAggregatorEndpoint to automatically call CreateSignatureAggregatorInstance

Reply:
Better to keep them separated. I made it so it will retry https://github.com/ava-labs/avalanche-cli/pull/2865

Reply:
ok will add the comment there